amplify/backend/function/risestandstrongcb34053a/src/routes/announcements.js

```diff
@@ -129,8 +131,19 @@ router.post('/', async (req, res) => {
  *         "404":
  *            description: Announcement with the specified announcementId does not exist
  */
+
 router.put('/:announcementId', async (req, res) => {
-    res.end();
+    const data = req.body;
+    if (!AWS_Auth.Auth.currentAuthenticatedUser()){
+        res.status(403)
```

Comment:
res.status() only set the status of the response to the input it doesn't actually send the response